src/test/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbTemplateIT.java

```diff
@@ -131,10 +131,10 @@ public void testFindByIdPartition() {
     }
 
     @Test
-    public void testUpdate() {
+    public void testUpsert() {
```

Comment:
add test to test insert new document with upsert() method?

Reply:
Because update method is removed.